backend/functions/README.md

```diff
@@ -0,0 +1,56 @@
+# CAMS Backend Functions
+
+## Node.js Version
+
+[Node Version Manager](https://github.com/nvm-sh/nvm) supports consistent use of a specific version of Node.js across environments.
+
+### Install `nvm`
+
+See: https://github.com/nvm-sh/nvm#install--update-script
+
+### `.nvmrc`
+
+The Node.js version used by the project is specified in the `.nvmrc` resource file.
+
+### Installing a Node.js version
+
+Use `nvm` to install a Node.js version.
```

Comment:
@jamesobrooks @btposey @magile32 @amorrow-flexion think we should update the build on USTP ADO as well to match this?

Reply:
This is an opportunistic improvement without any urgent need to be merged. No JIRA ticket, so one can be added to track this. Commit messages can be modified to reference the ticket if and when one is created.

The node version of the Azure function runtime in use should drive the version in the "backend" directory. The opposite may be true with the React app. The Github runner can/should be able to use NVM to use the specified node version to build the UI bundle.